src/test/java/uk/gov/justice/hmpps/datacompliance/services/events/listeners/OffenderPendingDeletionListenerTest.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.justice.hmpps.datacompliance.services.events.listeners;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class OffenderPendingDeletionListenerTest {
+
+    private OffenderPendingDeletionListener listener;
+
+    @BeforeEach
+    void setUp() {
+        listener = new OffenderPendingDeletionListener(new ObjectMapper());
```

Comment:
You could probably just inline this above.  I guess there will be more setup to come though...